occurrences/tests/test_api.py

```diff
@@ -1442,10 +1442,15 @@ def test_enrol_invalid_group_size(
 # The auto_acceptance calls Enrolment.approve,
 # so it needs to be tested with both the boolean values.
 @pytest.mark.parametrize("auto_acceptance", [True, False])
+@pytest.mark.parametrize(
+    "is_multi_enrolment",
+    [True, False],
```

Comment:
Not necessary to do (i.e. you can leave this as is if you want to): Removing the trailing comma and rerunning black would fit this on a single line.